engines/hypno/detection.cpp

```diff
@@ -0,0 +1,156 @@
+/* ScummVM - Graphic Adventure Engine
+ *
+ * ScummVM is the legal property of its developers, whose names
+ * are too numerous to list here. Please refer to the COPYRIGHT
+ * file distributed with this source distribution.
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ *
+ */
+
+#include "base/plugins.h"
+#include "common/translation.h"
+#include "engines/advancedDetector.h"
+#include "hypno/hypno.h"
+
+static const DebugChannelDef debugFlagList[] = {
+	{Hypno::kHypnoDebugMedia, "media", "Media debug channel"},
+	{Hypno::kHypnoDebugParser, "parser", "Parser debug channel"},
+	{Hypno::kHypnoDebugScene, "scene", "Scene debug channel"},
+	{Hypno::kHypnoDebugArcade, "arcade", "Arcade debug channel"},
+	DEBUG_CHANNEL_END};
+
+namespace Hypno {
+static const PlainGameDescriptor hypnoGames[] = {
+	{"sinister-six", "Marvel Comics Spider-Man: The Sinister Six"},
```

Comment:
In general, we try to avoid use of a dash (`-`) in the target names as that is used as a separator between the generated target name, e.g. `monkey-amiga-de`. Thus, it would be better to switch to `sinistersix` and `soldierboyz` respectively.